client/related_items/RelatedItems.jsx

```diff
@@ -1,11 +1,36 @@
 import React from 'react';
 import ItemsList from './ItemsList.jsx';
+import helpers from './helpers.js';
 
-const RelatedItems = (props) => (
-  <div>
-    <ItemsList />
-    <ItemsList />
-  </div>
-);
+//Then request Info + Styles for each of the product IDs
+//Manipulate data in server to only return what is needed here
+//Pass data from here down to the related products item list
+
+//Need to do something else for the Outfits list (local storage, cookies?)
+class RelatedItems extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      relatedItemsIds: []
+    };
+  }
+
+  componentDidMount() {
+    helpers.getRelatedItems(this.props.product_id)
+      .then(relatedItems => this.setState({ relatedItemsIds: relatedItems }));
+  }
+
+  render() {
+    return (
+      <div>
+        <h3>Related Items</h3>
+        <ItemsList listType={'relatedItems'} items={this.state.relatedItemsIds} />
```

Comment:
Do you need to have a string inside jsx? You might be able to remove the jsx and put the string in double quotes. Something to consider!

Reply:
Good catch! Does not need JSX - I'll merge this and update that in my next PR, thank you.